text_watermark.go

```diff
@@ -22,14 +22,24 @@ func drawText(p *creator.Paragraph, c *creator.Creator) {
 	p.SetColor(creator.ColorRGBFromHex("#" + color))
 	p.SetAngle(angle)
 
+	// Encountering problem with tiles and text watermark. Contributions welcome.
```

Comment:
for some reason, the tiles in paragraph isn't working fine. Generates error after couple of iterations:
> [DEBUG]  contentstream.go:94 ERROR: Name too long (Font100000000...
@ajaxray can you check if you can make it work? Can be targeted outside the scope of this PR.